nemo/collections/nlp/models/language_modeling/megatron_gpt_prompt_learning_model.py

```diff
@@ -465,6 +465,9 @@ def on_train_end(self):
             self.add_ptuned_prompts_to_prompt_table()
             logging.info(f"All p-tuned prompts where moved to the prompt table.")
 
+        self.virtual_prompt_style = 'inference'
+        self.virtual_prompt_source = 'prompt-table'
+
```

Comment:
for better practice, let's use `enum` types instead of string for virtual_prompt_style and virtual_prompt_source.

Reply:
They both work fine.

Reply:
Like this?

Reply:
something like this
```python
import enum

class VirtualPromptStyle(enum.Enum):
    prompt_tuning = 0
    inference = 1

self.virtual_prompt_style = VirtualPromptStyle.inference
```